modules/aleph/src/Aleph/Entity/AlephRequest.php

```diff
@@ -249,10 +257,10 @@ public function getRequestType() {
   /**
    * Set the request date.
    *
-   * @param string $request_date
+   * @param \DateTime $request_date
    */
   public function setRequestDate($request_date) {
-    $this->requestDate = $request_date;
+    $this->requestDate = $request_date->format('Y-m-d');
```

Comment:
How about using `ALEPH_DATE` here?

Reply:
It should also be OK to specify a proper interface as the parameter type e.g.

```php
public function setRequestDate(\DateTimeInterface $request_date) {
```

The interface is preferable to the class so that both DateTime and DateTimeImmutable can be type hinted for.